.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+In order to efficiently and accurately address your issue or feature request, please read through the template below and answer all relevant questions. Your additional work here is greatly appreciated and will help us respond as quickly as possible. Please delete any sections or questions below that do not pertain to this request.
+
+For general support or usage questions, please use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com.).
+
+### Description
+
+Description of the bug or feature request and why it's a problem. Consider including:
+
+- The use case or overall problem you're trying to solve
+- Information about when the problem started
+
+### Prerequisites
+
+- [ ] I have checked the [README documentation](https://github.com/auth0/auth0-oidc-client-net/blob/master/README.md).
+- [ ] I have checked the [Auth0 Community](https://community.auth0.com/) for related posts.
+- [ ] I have checked for related or duplicate [Issues](https://github.com/auth0/auth0-oidc-client-net/issues) and [PRs](https://github.com/auth0/auth0-oidc-client-net/pulls).
+- [ ] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md).
+- [ ] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
+- [ ] I am reporting this to the correct repository.
+
+### Environment
+
+Please provide the following:
+
+- Version of the library being used
```

Comment:
Can you update this section to be specific for this library?

Reply:
That's just the version of this library right?  Or is there something more specific?

Reply:
I made it explicit in the other ones, like "Version of `auth0-oidc-client-net` used"

Reply:
Okay but I feel the more these templates diverge the more difficult it is going to be to deploy future updates to the templates.

Reply:
The more specific the questions here, the more clear the answers. Asking for versions of specific things rather than "versions of things that might be relevant" will be more effective, IMHO. It's how all the other templates were submitted.